src/TimeSelect.jsx

```diff
@@ -15,9 +15,12 @@ var TimePicker = React.createClass({
     value: React.PropTypes.instanceOf(Date),
     start: React.PropTypes.number,
     end: React.PropTypes.number,
+    time: React.PropTypes.object,
     step: React.PropTypes.number,
     locale: React.PropTypes.string,
     onChange: React.PropTypes.func,
+    minutesHoursChanged: React.PropTypes.func,
```

Comment:
alternatively could we be slightly cleverer and refactor onChange to also work when `separateHourMins` is true? <-- I'd prefer this option

You could do something like:
```
const onChange = ( { hours, minutes} ) => { // blah }

<TimeSelect onChange={onChange} time={ { hours: 12, minutes: 30 } } />
```

then in TimeSelect:
```
changeMinutes = ( e ) => this.props.onChange({
  hours: this.props.time.hours,
  minutes: e.target.value
})

changeHours = ( e ) => this.props.onChange({
  hours: e.target.value
  minutes: this.props.time.value
})

changeTime = ( e ) => {
  const hourMins = e.target.value.split(':')
  return this.props.onChange({
    hours: hourMins[0],
    minutes: hourMins[1]
  })
}

<HourInput onChange={changeHours} />
<MinuteInput onChange={changeMinutes} />
<TimeInput onChange={changeTime} />
```
(see further comments on HourInput and MinuteInput)